velox/functions/prestosql/aggregates/MinMaxByAggregates.cpp

```diff
@@ -32,13 +32,18 @@ constexpr bool isNumeric() {
       std::is_same_v<T, float> || std::is_same_v<T, double>;
 }
 
+template <typename T>
+constexpr bool isDate() {
+  return std::is_same_v<T, Date>;
+}
+
 template <typename T, typename TAccumulator>
 void extract(
     TAccumulator* accumulator,
     const VectorPtr& vector,
     vector_size_t index,
     T* rawValues) {
-  if constexpr (isNumeric<T>()) {
+  if constexpr (isNumeric<T>() or isDate<T>()) {
```

Comment:
Looks like isNumeric and isDate are always used together. Consider replacing these with a single function isNumericOrDate.

Reply:
Thanks for the suggestion, reverted back to single function isNumericOrDate.